tests/Makefile.am

```diff
@@ -148,6 +149,12 @@ test_update_broken_delta_LDFLAGS = $(test_ldflags)
 test_update_broken_delta_LDADD = $(test_ldadd)
 test_update_broken_delta_SOURCES = test-update-broken-delta.c
 
+test_update_refspec_checkpoint_CPPFLAGS = $(test_cppflags)
+test_update_refspec_checkpoint_CFLAGS = $(test_cflags)
+test_update_refspec_checkpoint_LDFLAGS = $(test_ldflags)
+test_update_refspec_checkpoint_LDADD = $(test_ldadd)
+test_update_refspec_checkpoint_SOURCES = test-update-refspec-checkpoint.c
```

Comment:
This new file seems to be missing from the commit.

Reply:
Yikes, thanks for catching. I'll add it now.